src/main/java/org/example/web/MainServlet.java

```diff
@@ -9,24 +9,27 @@
 import org.example.model.Arguments;
 import org.example.model.JspAttribute;
 import org.example.model.Response;
+import org.example.service.DeveloperService;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.sql.SQLException;
 import java.util.List;
 
 public class MainServlet extends HttpServlet {
 
+    private static final String FORM_PAGE_URL = "/DeveloperApi/form";
```

Comment:
Is it possible to replace by UrlPath enum?

Reply:
fixed